extras/ribbon.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2015 Petr Shevtsov (petr.shevtsov@gmail.com)
+ * MIT-licensed (http://opensource.org/licenses/MIT)
+ *
+ * Plugin options:
+ *
+ *  `data`: Array of numeric values (0-1) corresponding to the position
```

Comment:
Why the palette / data distinction? Why not just pass in a list of colors for each rectangle?